routes/user/auth.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const {
+  forgotPasswordAction,
+  newPassword,
+  newPasswordAction,
+  signup,
+} = require("../../controllers/auth");
+const authRouter = express.Router();
+
+authRouter.post("/register", signup);
+authRouter.post("/forgotPassword", forgotPasswordAction);
+// new password
+authRouter.get("/newpassword/:id/:token", newPassword);
```

Comment:
what is the difference between these two routes?
why naming not according to functions